kuksa_databroker/databroker/tests/read_write_values.rs

```diff
@@ -90,7 +90,22 @@ async fn a_known_data_entry_has_value(
     value: String,
 ) {
     set_value(w, value_type, path, data_type, value).await;
-    w.assert_set_response_has_succeeded()
+    w.assert_set_succeeded()
+}
+
+#[when(expr = "a client uses a token with auhtorization {word}")]
```

Comment:
in the Token struct, what is called *authorization* here, is called *scope* instead. Using common language across the two places will make it easier to read/understand.